drivers/nrf_wifi/fw_if/umac_if/inc/fmac_util.h

```diff
@@ -66,6 +66,27 @@
 #define NRF_WIFI_FMAC_IPV6_TOS_SHIFT 0x04 /* 4bit */
 #define NRF_WIFI_FMAC_ETH_TYPE_MASK 0xFFFF
 
+#if defined(NRF70_PROMISC_DATA_RX)
```

Comment:
Since these are not `generic` utility type structures defines/structures etc. I would propose having them in a separate header file specific to `promiscuous` mode.